notebooks/templates/ai_platform_notebooks_template.ipynb

```diff
@@ -1,524 +1,634 @@
 {
```

Comment:
<p>Turn this into a complete sentence? And it should say "may be able".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>done</p>